remi/gui.py

```diff
@@ -537,7 +538,7 @@ def set_on_mousedown_listener(self, listener, funcname):
     def onmouseup(self, x, y):
         """Called when the user releases left or right mouse button over a Widget.
         """
-        return self.eventManager.propagate(self.EVENT_ONMOUSEUP, [x, y])
+        return self.eventManager.propagate(self.EVENT_ONMOUSEUP, (x, y))
```

Comment:
Yeah, I've not passed yet userdata in other callbacks, it is just implemented for set_on_click_listener. The change you highlighted is just a conversion from list to tuple and is not directly related to the userdata. Is this clear?